com.unity.multiplayer.mlapi/Runtime/Core/NetworkManager.cs

```diff
@@ -388,14 +390,23 @@ private void Initialize(bool server)
 
             SnapshotSystem = new SnapshotSystem();
 
-            //This 'if' should never enter
             if (NetworkTickSystem != null)
             {
-                NetworkTickSystem.Dispose();
+                NetworkTickSystem.Tick -= OnNetworkManagerTick;
                 NetworkTickSystem = null;
             }
 
-            NetworkTickSystem = new NetworkTickSystem(NetworkConfig.NetworkTickIntervalSec);
+            if (server)
+            {
+                NetworkTimeSystem = NetworkTimeSystem.ServerTimeSystem();
+            }
+            else
+            {
+                NetworkTimeSystem = new NetworkTimeSystem(1d / NetworkConfig.TickRate, 1d / NetworkConfig.TickRate, 0.2);
```

Comment:
For readability, can we have `1.0 /`? Seems silly, but at first, I parsed 1d as 1 day, in the context of time. Similarly, 0.2 could be in a variable with a name, otherwise, as we glance, it's not obvious what it is. All minor stuff.

Reply:
Adjusted to use `1.0` instead of `1d` for the `0.2` I expect people to use an IDE like Rider or VS which writes parameter names in front of the values.